spec/rails_helper.rb

```diff
@@ -67,9 +67,9 @@ class Analytics
   end
 
   if !ENV['CI']
-    config.before(:all, js: true) do
+    config.before(:suite, js: true) do
```

Comment:
Just noticed this warning when run locally:

```
WARNING: `:suite` hooks do not support metadata since they apply to the suite as a whole rather than any individual example or example group that has metadata. The metadata you have provided ([{:js=>true}]) will be ignored.
```

Unfortunate, since it's rather wasteful to run the Webpack build if not concerned with JavaScript-enabled feature specs 🤔  I wonder if there's another way to apply it at per-spec/group level, but only run once. Global var? 😬 

```suggestion
    config.before(js: true) do
      next if defined?($ran_webpack_build)
      $ran_webpack_build = true
```

Reply:
After some testing, the above suggestion does seem to work as expected. Obviously globals are gross, so happy to apply any alternative ideas, but otherwise will move forward with this.

Reply:
Yeah this seems fine for now, better to fix the issue